jupyter_kernel_mgmt/subproc/manager.py

```diff
@@ -127,9 +117,10 @@ def signal(self, signum):
                 return
             except OSError:
                 pass
-        self.kernel.send_signal(signum)
+        await maybe_future(self.kernel.send_signal(signum))
```

Comment:
kill and send_signal are also synchronous with the asyncio Process class - docs: https://docs.python.org/3/library/asyncio-subprocess.html#asyncio.asyncio.subprocess.Process.send_signal

Reply:
Yeah, I guess I had my mind on EG's process proxies where Popen is abstracted.  I'm update this.